array_analyzer/workflows/registration_workflow.py

```diff
@@ -96,20 +98,43 @@ def point_registration(input_dir, output_dir):
             stop_criteria=0.1,
             debug=constants.DEBUG,
         )
+
+        # Warn if fit is still bad
         if min_dist > constants.REG_DIST_THRESH:
-            warnings.warn("Registration failed, repeat with outlier removal")
+            spot_coords, im_filtered = img_processing.get_spot_coords(
+                image,
+                min_area=200,
+                min_thresh=100,
+                max_thresh=255,
+                min_circularity=0.1,
+                min_convexity=0.5,
+                min_dist_between_blobs=10,
+                min_repeatability=2,
+                sigma_gauss=10,
```

Comment:
So the spot detection consistently fails for small spots? In a future PR I could try combining different scales in spot detection, but this is good for now...

Reply:
Yes this is more a temporary fix because I didn't want to change the parameters you have tuned for spot detection to work for cuttlefish data, but this can be done inside spot detection

Reply:
I did another iteration on the fishy registration PR to increase stability for the May 2 datasets. Sorry for making your changes out of sync! But I did find that smaller sigma worked better for most data thanks to your changes so thank you.